### PR TITLE
Timers for Recent Blocks + Recent Transactions

### PR DESCRIPTION
**Pull request by robin-thomas**
* Timers for Recent Blocks + Recent Transactions
* Switch between testnet dropdown options

### CODE REVIEW

File: components/dashboard/DashboardTransactionsRow.js [hunk 1]
```diff
@@ -49,86 +53,120 @@ const TransactionImage = {
...
+
+  componentDidMount() {
+    this.timer = setInterval(() => this.tick(), 1000);
+  }
+
+  componentWillUnmount() {
+    clearInterval(this.timer);
+  }
+
+  tick() {
```
Comment (frol) [inline components/dashboard/DashboardTransactionsRow.js]: I think we can avoid code duplication with a separate component, which will handle the ticks.

File: package.json [hunk 2]
```diff
@@ -20,6 +20,7 @@
   "homepage": "https://github.com/nearprotocol/near-explorer#readme",
   "dependencies": {
     "bootstrap": "^4.3.1",
+    "moment": "^2.24.0",
```
Comment (frol) [inline package.json]: FYI, Moment.js [used to be quite fat](https://github.com/moment/moment/issues/3376) the last time I checked it. [Luxon](https://moment.github.io/luxon/docs/manual/why.html) may be a better alternative, but I haven't checked it myself.

I think we can live with Moment.js for now, but we should keep this point in mind once.

File: components/HeaderDropdownNav.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+class HeaderDropdownNav extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      network: "Testnet One"
+    };
+
+    this.selectNetwork = this.selectNetwork.bind(this);
```
Comment (frol) [inline components/HeaderDropdownNav.js]: It seems easier to define the methods with arrow functions and avoid this manual rebinding. So, you just drop this line and define the method as:

```
selectNetwork = (e) => {
  ...
}
```

Just a random article on this topic: https://javascriptweblog.wordpress.com/2015/11/02/of-classes-and-arrow-functions-a-cautionary-tale/

File: package.json (hunk 2 above)
Comment (robin-thomas) [reply package.json]: I'm pretty used to `Moment.js`. So used it. But yes, will take a look into that later.

File: components/dashboard/DashboardTransactionsRow.js (hunk 1 above)
Comment (robin-thomas) [reply components/dashboard/DashboardTransactionsRow.js]: Yes, I did think of that initially. Anyway, have created a new Timer component to handle the ticks.

File: components/HeaderDropdownNav.js (hunk 3 above)
Comment (robin-thomas) [reply components/HeaderDropdownNav.js]: Thanks. Switched to that.

**Pull request merged**